searches.csv

```diff
@@ -0,0 +1,97 @@
+Timestamp,Number of results,Location,Radius,Keyword,Minimum Salary,Maximum salary,Working pattern,Phase,NQT,,

```

Comment:
I can't see this file being used anywhere, did you mean to commit it?

Reply:
Whoops! Yeah, I'll ding that